plugin.rb

```diff
@@ -16,9 +16,10 @@
 
 enabled_site_setting :discord_enabled
 
-class DiscordAuthenticator < ::Auth::OAuth2Authenticator
```

Comment:
To make this switch seamless, you also need to add a database migration. For example: https://github.com/discourse/discourse-oauth2-basic/blob/master/db/migrate/20190724055909_move_to_managed_authenticator.rb